lib/python/qmk/path.py

```diff
@@ -3,6 +3,7 @@
 import logging
 import os
 from pathlib import Path
+from argparse import FileType as _FileType
```

Comment:
Github won't let me suggest it, but this should go on line 3 as well to keep everything nicely ordered.

```suggestion
import argparse
```